packages/cli/cli/src/commands/generate/generateAPIWorkspace.ts

```diff
@@ -59,6 +59,9 @@ export async function generateWorkspace({
             context
         });
     } else {
+        if (!token) {
+            return context.failAndThrow("Must provide token if 'useLocalDocker' is false");
```

Comment:
@dsinghvi what are your thoughts on this?
```suggestion
            return context.failAndThrow("Please run fern login. No token was found, and `--local` is not enabled.");
```

Reply:
Not sure if "no token was found" or "no token was provided" makes more sense here.